shared/src/business/entities/EntityConstants.ts

```diff
@@ -210,6 +218,15 @@ export const DOCUMENT_SERVED_MESSAGES = {
   SELECTED_CASES: 'Document served to selected cases in group.',
 };
 
+export const VALIDATION_ERROR_MESSAGES = {
```

Comment:
Are these validation error messages only used for JudgeActivityReportSearch? If so, I think I would prefer they live in that entity since EntityConstants is already getting pretty crowded and no one else needs to know about them.

Reply:
Yeah its only used only for [JudgeActivityReportSearch.ts](https://github.com/ustaxcourt/ef-cms/pull/3799/files/96a9b0554976f8cb2c91b92a04349e86bfdda5bd#diff-5495bccafdb9396bb5d4653ecd9f9af032d074bc54f330583627f215fb754b2b). I can move it. No problem

Reply:
Thanks!